universal-sentence-encoder/src/tokenizer/trie.ts

```diff
@@ -18,67 +18,29 @@
 import {stringToChars} from '../util';
 
 // [token, score, index]
-type OutputNode = [string[], number, number];
+// TODO fix typing
+// type OutputNode = [string[], number, number];
+type OutputNode = any; 
```

Comment:
Why doesn't the original typing work?

Reply:
Typescript complains that `this.word = [[], 0, 0];` doesn't have types; I'm not sure how to add them.

```
src/tokenizer/trie.ts:35:18 - error TS2739: Type '[]' is missing the following properties from type '[string[], number, number]': 0, 1, 2

35     this.word = [[], 0, 0];
```

Reply:
You need to change `public word: OutputNode[];` (line 29) to `public word: OutputNode;`

Looks like there are also two lint issues on lines 80, 84 - I think you just need to add curly braces around the if statements.

Reply:
Ah, that makes sense.